runtime/syntax/nftables.yaml

```diff
@@ -1,28 +1,25 @@
 filetype: nftables
 
 detect:
-    filename: "nftables.conf$"
+    filename: "((nftables.conf$)|(nftables.rules.d/)|(nftables.d/)|(nftables.rules$))"
     header: "^(#!.*/(env +)?nft( |$)|flush +ruleset)"
-    
+
 rules:
     - type: "\\b(chain|counter|map|rule|ruleset|set|table)\\b"
     - type: "\\b(ether|inet|i(cm)?p(x|(v?(4|6))?)|tcp|udp)\\b"
-    - special: "\\b(elements|hook|policy|priority|type)\\b"
-    - identifier: "\\b(ct|iif|iifname|meta|oif|oifname|th)\\b"
-    - statement: "\\b(accept|drop|goto|jump|log|masquerade|reject)\\b"
+    - special: "\\b(elements|hook|policy|priority|type|state)\\b"
+    - identifier: "\\b(ct|iif|iifname|meta|oif|oifname|th|dport|sport|saddr|daddr)\\b"
+    - statement: "\\b(accept|drop|goto|jump|log|masquerade|reject|limit|queue)\\b"
     - preproc: "\\b(add|define|flush|include|delete)\\b"
-    - symbol: "([-=/:;,@]|[!=])"
-    - symbol.operator: "[<>.&|^!]|\\b(and|ge|gt|le|lt|or|xor)\\b"
-    - constant.string: '([\"]{1})(.*)([\"]{1})'
-    - constant.string: 
-         start: "(['\"])" 
-         end: "(['\"])" 
+    - symbol.operator: "[<>.&|^!=:;,@]|[!=]|\\b(and|ge|gt|le|lt|or|xor)\\b"
```

Comment:
We've now `!=` twice, right?

Reply:
Oh ups. i didn't noticed it. removed the first one :-).